src/index.js

```diff
@@ -48,13 +56,16 @@ const displayTasks = () => {
 
   const tasks = getTasks();
 
-  // eslint-disable-next-line no-unused-vars
-  tasks.forEach((task, index) => {
+  tasks.forEach((task) => {
     const div = document.createElement('div');
     div.classList.add('task');
     div.setAttribute('id', `task-${task.index}`);
-    div.innerHTML = `<img src="${uncheckIcon.src}" alt="Uncheck Icon">
-                     <p contenteditable>${task.description}</p>
+    const checkboxClass = task.completed ? 'checkico' : 'uncheckico';
```

Comment:
- [ ] Great job adding the interactive functionality. As demanded by project requirements, kindly add a new js file and move all code related to the complete functionality so you can import it as module.